components/server/src/user/token-service.ts

```diff
@@ -29,6 +31,17 @@ export class TokenService implements TokenProvider {
 
     async getTokenForHost(user: User | string, host: string): Promise<Token | undefined> {
         const userId = User.is(user) ? user.id : user;
+
+        // EXPERIMENT(sync_refresh_token_exchange)
+        const syncRefreshTokenExchange = await getExperimentsClientForBackend().getValueAsync(
+            "sync_refresh_token_exchange",
+            false,
+            {},
```

Comment:
Should we add host as a property of Feature Flag? Just in case they have multiple SCM

Reply:
We are interested to role this out for all, so I think we should test it as such. The difference between SCMs should be properly abstracted behind the Token abstraction.